### PR TITLE
Minor changes in vtk reference guide [doc-build]

### PR DESCRIPTION
**Pull request by xavArtley**
Some changes in the reference guide of the VTK panel when using native vtk object
+ remove reference to altair

### CODE REVIEW

File: examples/reference/panes/VTK.ipynb [hunk 1]
```diff
@@ -21,7 +21,6 @@
     "For layout and styling related parameters see the [customization user guide](../../user_guide/Customization.ipynb).\n",
     "\n",
     "* **``camera``** (dict): A dictionary reflecting the current state of the VTK camera\n",
-    "* **``object``** (dict or altair Chart): Either a dictionary containing a vega or vega-lite plot specification or an altair Chart\n",
```
Comment (philippjfr) [inline examples/reference/panes/VTK.ipynb]: This was of course a stupid copy/paste error but object should definitely still be mentioned here.

**Comment by xavArtley** [comment]
In the same time height of the panel is not respected on the web site
image
whereas it is on my live notebook
image

**Comment by philippjfr** [comment]
Also while you're at it, could you change the ``sizing_mode='stretch_both`` to stretch_width.

**Comment by philippjfr** [comment]
Ha, good timing, I'm hoping that stretch_width will fix the website sizing issue.

**Comment by philippjfr** [comment]
Thanks looks great, I'll start a doc build as soon as this is merged.

**Pull request merged**